doc/getting-started.md

```diff
@@ -123,7 +123,7 @@ For example, to check it's working:
 
 ```
 $ curl -X 'GET' \
-  'http://localhost:8001/latest/me' \
+  'http://localhost:8001/latest/whoami' \
   -H 'accept: application/json' \
   -H 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJib2IifQ.KHkILtsJaCmueOfFCj79HGr6kHamuZFdB1Yz_5GqcC4'
 {"_id":"615f30020eb7c3c6616e5ac3","username":"bob","hashed_password":"$2b$12$VtfVij6zz20F/Qr0Ri18O.11.0LJMMXyJxAJAHQbKU0jC96eo2fr.","active":true}
```

Comment:
Could you please update the sample command result as well? I should have fixed it as a part of https://github.com/kernelci/kernelci-api/pull/280 but I missed it. Since you are updating documentation for the same endpoint, I'd appreciate if you could add another commit with suggested change.
```suggestion
{"id":"615f30020eb7c3c6616e5ac3","username":"bob","hashed_password":"$2b$12$VtfVij6zz20F/Qr0Ri18O.11.0LJMMXyJxAJAHQbKU0jC96eo2fr.","active":true}
```

Reply:
Sure thing! Thank you for pointing it out for me, I missed this change while setting up my local instance.

Reply:
Thank you.

Reply:
Resolved in https://github.com/kernelci/kernelci-api/pull/297/commits/66dc4252cacb22e570002455c55f5e844f3f61bc